src/test/java/seedu/fma/storage/JsonSerializableLogBookTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.fma.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.fma.testutil.Assert.assertThrows;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.fma.commons.exceptions.IllegalValueException;
+import seedu.fma.commons.util.JsonUtil;
+import seedu.fma.model.AddressBook;
+import seedu.fma.testutil.TypicalLogs;
+
+public class JsonSerializableLogBookTest {
+
+    private static final Path TEST_DATA_FOLDER = Paths.get("src", "test", "data", "JsonSerializableLogBookTest");
+    private static final Path TYPICAL_PERSONS_FILE = TEST_DATA_FOLDER.resolve("typicalPersonsAddressBook.json");
+    private static final Path INVALID_PERSON_FILE = TEST_DATA_FOLDER.resolve("invalidPersonAddressBook.json");
+    private static final Path DUPLICATE_PERSON_FILE = TEST_DATA_FOLDER.resolve("duplicatePersonAddressBook.json");
+
+    @Test
+    public void toModelType_typicalPersonsFile_success() throws Exception {
```

Comment:
Person -> Log

Reply:
Remember to change the method name to match Logs. This is repeated for the other tests in this file